dist/pom.xml

```diff
@@ -90,7 +221,7 @@
                                 <mkdirs>
                                     <paths>
                                         <path>etc/sds/crossdata</path>
```

Comment:
Given that paths are absolute, I'll add the "/" at the start, just in case and in order to clarify the code